bitManipulation/missingNumberInArray.java

```diff
@@ -0,0 +1,18 @@
+package bitManipulation;
```

Comment:
// https://leetcode.com/problems/missing-number/

package bitManipulation;

Reply:
Okk will do them. Also can you tell if this repo has been excluded from Hacktoberfest 2022?

Reply:
I have done the changes